latex/utxo.tex

```diff
@@ -232,21 +231,21 @@ \subsection{UTxO Transitions}
 
 The accounting for the reward withdrawals is not done in this transition system.
 The rewards are tracked with the delegation state and will
-be removed in the final delegation transition, see ~\ref{eq:delegs-base}.
+be removed in the final delegation transition, see Figure~\ref{eq:delegs-base}.
```

Comment:
this refers to an equation, not a figure

Reply:
oops! thanks